Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    json (2.1.0)
-    jwt (1.5.6)
-    jwt_signed_request (2.4.0)
-      jwt (~> 1.5.0)
+    json (2.2.0)
+    jwt (2.1.0)
+    jwt_signed_request (2.5.1)
+      jwt (>= 1.5.0, < 2.2.0)
       rack
     loofah (2.2.3)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.1)
-    mini_portile2 (2.3.0)
+    method_source (0.9.2)
+    mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.1)
+    money (6.13.2)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.8.5)
-      mini_portile2 (~> 2.3.0)
+    nokogiri (1.10.1)
```

Comment:
### nokogiri

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.8.5 → 1.10.1

[_~~change-log~~, [source-code](https://github.com/sparklemotion/nokogiri)_]





<details>
<summary>Commits</summary>

A change of **151** commits. See the full changes on [the compare page](https://github.com/sparklemotion/nokogiri/compare/v1.8.5...v1.10.1).

These are the first 10 commits:
- (d92d1c7) [Add SemVer compatibility and gem version badges to README](https://github.com/sparklemotion/nokogiri/commit/d92d1c7b0bb2b3daa19aebb1215b5dd3c49cb8fa)
- (859845b) [Remove Version Eye badge](https://github.com/sparklemotion/nokogiri/commit/859845b9bd226014292db94c8c00fcb635e66cb4)
- (f901d8b) [Merge pull request #1811 from yasslab/remove_versioneye_badg…](https://github.com/sparklemotion/nokogiri/commit/f901d8bd1d124836491a3bff69cf255590d94ef8)
- (801eacb) [update to latest hoe-debugging](https://github.com/sparklemotion/nokogiri/commit/801eacb44863c06645adc7f1a6152d0ffa855ea0)
- (0618cce) [concourse: valgrind builds have TESTOPTS=-v](https://github.com/sparklemotion/nokogiri/commit/0618ccec3245778460d3a59d6ce828d940a70dcd)
- (ec52483) [valgrind suppression file for false positives](https://github.com/sparklemotion/nokogiri/commit/ec52483c4c5230b9e3e47b991cc302bcb9b992bb)
- (48dfc50) [allow rbx to fail for now](https://github.com/sparklemotion/nokogiri/commit/48dfc502c007df26e7ccaa9a01fac2cf84a48df3)
- (9498d43) [concourse: use valgrind suppressions from the ci resource](https://github.com/sparklemotion/nokogiri/commit/9498d433b75b1726b5dc6aad212b4efcd8cc3399)
- (dd1496d) [concourse:valgrind: fix naming of supression file](https://github.com/sparklemotion/nokogiri/commit/dd1496d3186112657df1f1d1352bed951910b2d8)
- (d13284a) [xml_namespace.c: avoid multiple calls to Data_Wrap_Struct](https://github.com/sparklemotion/nokogiri/commit/d13284abbd8e3aa77ab899e7b9ca514b84bdd7dd)


</details>